CMakeLists.txt

```diff
@@ -498,6 +498,40 @@ IF (ENABLE_GCOV)
   ENDIF()
 ENDIF(ENABLE_GCOV)
 
+# Require C++11 while leaving possibility to fall back by modifying this file
```

Comment:
This code should be unnecessary with the updated minimum cmake version and the move to cmake mechanisms to specific the C++ standard.

Reply:
@mmorale3 : when I quickly scanned the updated cmakelists.txt it looked like it might actually be from an earlier version - are these updates actually need?

Reply:
Just remove this part. Everything is already on line 388.